src/pages/home/ReportScreen.js

```diff
@@ -340,6 +342,7 @@ class ReportScreen extends React.Component {
                                     isComposerFullSize={this.props.isComposerFullSize}
                                     isDrawerOpen={this.props.isDrawerOpen}
                                     parentViewHeight={this.state.skeletonViewContainerHeight}
+                                    policy={policy}
```

Comment:
Can we pass the policies directly and have the filter in ReportActionsView similar to how MoneyRequestHeader is doing?
https://github.com/Expensify/App/blob/619a5298771ba5e42ae6f1596a37e32b9cd16fc0/src/pages/home/ReportScreen.js#L297

The reason I saying is, for normal chats and group chats the policy would be undefined and we would be passing a undefined value, which won't be a good code practice. With the proposed way it will be in sync with existing code as well.

Reply:
@hungvu193

Reply:
Sure @abdulrahuman5196